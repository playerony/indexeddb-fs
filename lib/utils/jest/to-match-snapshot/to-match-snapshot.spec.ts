import { toMatchSnapshot } from '@utils';

describe('toMatchSnapshot Function', () => {
  toMatchSnapshot(jest.fn());

  it('should throw an error when passed parameter is not a function', () => {
    // @ts-ignore
    expect(() => toMatchSnapshot(null)).toThrow('parameter is not a function');
  });
});
