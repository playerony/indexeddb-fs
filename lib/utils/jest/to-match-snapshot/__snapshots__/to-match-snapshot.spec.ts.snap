// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`toMatchSnapshot Function snapshot test should match snapshot 1`] = `undefined`;
